### ISSUE TITLE
[MAP]: Concrete_Jungle

### ISSUE THREAD

**Issue by Mtlhead7** [Warzone2100/map-submission#116]
### Upload Map

[10c-Concrete_Jungle.wz.zip](https://github.com/user-attachments/files/16719700/10c-Concrete_Jungle.wz.zip)

### Authorship

Mine: I am the author of this map

### Map Description (optional)

```text
10 Player Skirmish
```

**Comment by vaut** [comment]
```
  "balance": {
      "structures": false,
```
please fix the buildings on the map

@past-due Is it possible to issue balance absence warnings automatically?
And I will ask back, is it really impossible to display a preview of the card in the issue itself? The current method is very annoying.

**Comment by Mtlhead7** [comment]
Fix attached.
Don

**Comment by Mtlhead7** [comment]
This email said the map passed.  Why isn't it on the database yet?
Don

**Comment by vaut** [comment]
The map still contains structural asymmetries and has not been approved by moderators.

**Comment by Mtlhead7** [comment]
What does that mean: "structural asymmetries"?
I don't code.  I just make maps in FlaME v 1.29.  I just need to know what
is wrong on the map in layman's terms and I will fix it.
Don

**Comment by vaut** [comment]
The number of buildings on the map is not the same for all players.

**Comment by Mtlhead7** [comment]
I sent the fix 5 days ago but here it is again.
Don

**Comment by vaut** [comment]
https://github.com/Warzone2100/map-submission/issues/121 Double

**Comment by Mtlhead7** [comment]
Hi,
I have been checking the database frequently but this map is still not up
there.  Is there a reason why? I thought everything was green.
Don
